### PR TITLE
Update: Added travis ci support for modern node versions

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,15 @@
-language: node_js
 sudo: false
+
+language: node_js
+
 node_js:
-    - "0.12"
+    - 6
```
Comment (nzakas) [inline .travis.yml]: Please put in increasing version number order

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
     - 4
+    - '0.12'
+
+cache:
```
Comment (nzakas) [inline .travis.yml]: Please remove this.

**Comment by amilajack** [comment]
Fixed

**Comment by nzakas** [comment]
Can you sign the CLA? Also, it looks like the Node 6 environment is failing, so this can't be merged into that is addressed. Can you investigate?

**Comment by dannyfritz** [comment]
@amilajack I took a brief look and my guess is that TypeScript is not installed automatically in 6.x because it is a peer dependency.

**Comment by dannyfritz** [comment]
This PR should be closed and not merged. #98 solved this.

**Comment by platinumazure** [comment]
Thanks @dannyfritz, closing per your comment.

**Pull request closed without merging**